tools/issue-labeler/CHECKLIST.md

```diff
@@ -0,0 +1,12 @@
+# Checklist for creating a new repo
```

Comment:
We'll want to remove all the MD files as well here.   These are also represented in the root.  _(these are hold overs from the .NET repo)_